spec/spec_helper.rb

```diff
@@ -14,9 +14,26 @@
   config.default_cassette_options = {:record => :none, :allow_unused_http_interactions => true}
   config.hook_into :webmock
 
-  Rails.application.secrets.autosde_defaults.keys.each do |secret|
-    config.define_cassette_placeholder(Rails.application.secrets.autosde_defaults[secret]) do
-      Rails.application.secrets.autosde[secret]
+  defaults = {
+    "appliance_host"        => "autosde-appliance-host",
+    "site_manager_user"     => "autosde",
+    "site_manager_password" => "change_me"
+  }
+  (Rails.application.credentials.autosde_defaults || defaults).each do |key, value|
+    config.define_cassette_placeholder(key) do
+      Rails.application.credentials.dig(:autosde, key) || value
```

Comment:
Note, I don't know exactly what it's doing here and realize we have the methods just below... but this needs to work in two scenarios:

1) User or CI with no Rails credentials defined:
  * no config/master.key, no config/credentials.yml.enc
  * Or based on env, no config/credentals/test.key or config/credentials/test.yml.enc
2) User or CI with rails credentials defined with the same format as seen here.

Reply:
In other words, if someone can run this with actual credentials to update cassettes, we can refactor it to minimize the duplication.  I didn't want to change more than I had to.

Reply:
Since we won't have `Rails.application.credentials.autosde_defaults` I think we can simplify this to just `defaults.each` the rest looks right :+1:  use a credentials value if it exists otherwise use the default value.